schemas/Manifest.yaml

```diff
@@ -7,8 +7,9 @@ properties:
   manifestItems:
     type: array
     example:
-      - "qr.example.com"
-      - "qr.test.com"
+      - "https://qr.example.com/i/123465678"
```

Comment:
Yes agree let's have a good example here, and possibly a link to the formatting / .well-known stuff? Using "/i" across all examples may be misleading, but I'm fine with it so long as there is documentation (want to make sure nobody accidentally hardcodes /i in their parser).

Reply:
Want to remind us that the goal here is the .well-known endpoint provides parameter serialization using a standard format: https://swagger.io/docs/specification/serialization/